homeassistant/components/zwave_mqtt/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain": "zwave_mqtt",
+  "name": "Z-Wave over MQTT",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/zwave_mqtt",
+  "requirements": [
+    "python-openzwave-mqtt==0.0.9"
+  ],
+  "after_dependencies": [
+    "mqtt"
+  ],
```

Comment:
Why is this an after dependency? This is a hard dependency at the moment

Reply:
I changed this to after deps to be able to catch the error in the config flow instead of just in the logs. Is there another option?

Reply:
@balloob what do you think? It's a better user experience to give feedback in the config flow. Do you still want to use a hard dependency?

Reply:
We talked about it and decided to keep it like this for now, but probably rework the dependency management for config flows later.